morphir/runtime/test/jvm/src/org/finos/morphir/runtime/EvaluatorMDMTests.scala

```diff
@@ -923,6 +970,50 @@ object EvaluatorMDMTests extends MorphirBaseSpec {
           )(
             Data.List.empty(Concept.Int32)
           )
+        ),
+        suite("sum")(
+          testEval("sum a list of ints returns sum of ints")("listTests", "listSumTest", List(1, 2))(
```

Comment:
some of your examples here don't match with the expected that you put in the elm section. I would keep them consistent

Reply:
Fair. It won't be perfect, since it tests with an empty list as well, but it the int, float, decimal ones can all be changed to match the elm expected (but in their own formats)

Reply:
thanks!